src/main/java/com/googlesource/gerrit/plugins/oauth/InitOAuth.java

```diff
@@ -35,11 +35,12 @@
   private final Section githubOAuthProviderSection;
   private final Section bitbucketOAuthProviderSection;
   private final Section casOAuthProviderSection;
+  private final Section gitlabOAuthProviderSection;
 
   @Inject
   InitOAuth(ConsoleUI ui,
-      Section.Factory sections,
-      @PluginName String pluginName) {
+            Section.Factory sections,
+            @PluginName String pluginName) {
```

Comment:
No formatting changes in the existing code. Pay attention: you shoudn't change any line in the existing code in this commit..